README.md

```diff
@@ -6,11 +6,13 @@ Notebook form. They help demonstrate not only the capabilities of
 important concepts and usage instructions.
 
 There are two main methods for setting up to run the recipes:
-- Use a prebuilt VS Code dev container (requires VS Code and Docker)
+
+1. [Use a prebuilt VS Code dev container](#run-in-a-dev-container)
```

Comment:
I'd prefer that we keep this unordered to prevent the impression that this is a sequence.